features/Extended Materials/Shaders/ExtendedMaterials/ExtendedMaterials.hlsli

```diff
@@ -311,45 +311,62 @@ namespace ExtendedMaterials
 		return coords;
 	}
 
-	// https://advances.realtimerendering.com/s2006/Tatarchuk-POM.pdf
+#if defined(TRUE_PBR)
+#	define SHADOW_INTENSITY 1.0
```

Comment:
PBR Parallax shadows now don't use the 2x multiplier. It makes the shadows (arguably) too strong with the PBR changes and I found no need for it to be there.

Reply:
Don't use defines for constants, there are actual constants for this

Reply:
skill issue moment